TheWarWithin/RogueSubtlety.lua

```diff
@@ -1472,6 +1464,7 @@ spec:RegisterAbilities( {
             -- applyBuff( "symbols_of_death_crit" )
 
             if legendary.the_rotten.enabled then applyBuff( "the_rotten" ) end
+            if talent.supercharger.enabled then addStack( "supercharged", nil, talent.supercharger.rank ) end
```

Comment:
Same thing about supercharge.

Reply:
Different same thing about supercharge.  Internally, use the same buff name consistently (and in the RegisterAura for `supercharged_combo_points`, add `copy = { "supercharge", "supercharged", "supercharger" }` so that all variants work from an APL perspective.